drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
Because CAN is also a separate stack. This group is defined in `sys/include/can/doc.txt`. There other opened PR (I 3 or 4) that will provide CAN hardware driver. I was unsure if it should go in the netdev_drivers.